config-solaris.md

```diff
@@ -0,0 +1,88 @@
+# Solaris Application Container Configuration
+
+Solaris application containers can be configured using the following properties, all of the below properties are the same as given in zonecfg(1M) man page, except milestone. The Solaris specification is entirely optional.
+
+## milestone
+The SMF(Service Management Facility) FMRI which should go to "online" state before we start the desired process within the container.
+
+**`milestone`** *(string, optional)*
+
+### Example
+```json
+"milestone": "svc:/milestone/container:default"
+```
+
+## limitpriv
+The maximum set of privileges any process in this container can obtain. The property should consist of a comma-separated privilege set specification as described in priv_str_to_set(3C) man page for the respective release of Solaris.
+
+**`limitpriv`** *(string, optional)*
+
+### Example
+```json
+"limitpriv": "default"
+```
+
+## max-shm-memory
+The maximum amount of shared memory allowed for this application container. A scale (K, M, G, T) can be applied to the value for each of these numbers (for example, 1M is one megabyte).
+
+**`max-shm-memory`** *(string, optional)*
+
+### Example
+```json
+"max-shm-memory": "512m"
+```
+
+## capped-cpu
+Sets a limit on the amount of CPU time that can be used by a container. The unit used translates to the percentage of a single CPU that can be used by all user threads in a container, expressed as a fraction (for example, .75) or a mixed number (whole number and fraction, for example, 1.25). An ncpu value of 1 means 100% of a CPU, a value of 1.25 means 125%, .75 mean 75%, and so forth. When projects within a capped container have their own caps, the minimum value takes precedence.
```

Comment:
Ok. Can you add the link to the PR itself?

On Fri, Apr 29, 2016 at 1:02 PM, Abhijeeth Nuthan <notifications@github.com>
wrote:

> In config-solaris.md
> <https://github.com/opencontainers/runtime-spec/pull/411#discussion_r61636723>
> :
>
> > +```json
> > +"limitpriv": "default"
> > +```
> > +
> > +## max-shm-memory
> > +The maximum amount of shared memory allowed for this application container. A scale (K, M, G, T) can be applied to the value for each of these numbers (for example, 1M is one megabyte).
> > +
> > +**`max-shm-memory`** *(string, optional)*
> > +
> > +### Example
> > +```json
> > +"max-shm-memory": "512m"
> > +```
> > +
> > +## capped-cpu
> > +Sets a limit on the amount of CPU time that can be used by a container. The unit used translates to the percentage of a single CPU that can be used by all user threads in a container, expressed as a fraction (for example, .75) or a mixed number (whole number and fraction, for example, 1.25). An ncpu value of 1 means 100% of a CPU, a value of 1.25 means 125%, .75 mean 75%, and so forth. When projects within a capped container have their own caps, the minimum value takes precedence.
>
> Both are actually one and the same , its just that structurally we expect
> them to be this way for legacy reasons and we can't really change the names
> or the way they are represented. The value of ncpus is the value of
> capped-cpu.
>
> Example: Set container to use 80% of cpu.
> "cappped-cpu" {
> "ncpus": "0.8"
> }
>
> Link to zonecfg(1M) manpage for Solaris 11.2:
> https://docs.oracle.com/cd/E36784_01/html/E36871/zonecfg-1m.html
> You can find all the properties explained. It changes for each release of
> Solaris, hence we ask user to refer to zonecfg(1M) for additional
> information.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/411/files/d57479d6f48fc50f584d566c26c9dc06da140946#r61636723>
>